packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -12,13 +12,13 @@ import {
   FUNDING_STRATEGY,
   INITIAL_BUDGET_AMOUNT
 } from '../constants';
-import {hexZeroPad} from 'ethers/utils';
 import {AddressZero} from 'ethers/constants';
 import * as firebase from 'firebase/app';
 import 'firebase/database';
 import debug from 'debug';
 import _ from 'lodash';
 const log = debug('web3torrent:payment-channel');
+const hexZeroPad = utils.hexZeroPad;
```

Comment:
Do you know why importing `hexZeroPad` is not compatible with storybook?

Reply:
Is this issue solved with your PR? Or it continues to happen?

Reply:
I left this comment before I dug into the root cause, so you can ignore the comment.